system/prometheus-vmware/values.yaml

```diff
@@ -0,0 +1,89 @@
+global:
+  domain: cloud.sap
+  clusterType: controlplane
+
+image:
+  repository: keppel.eu-de-1.cloud.sap/ccloud-dockerhub-mirror/prom/prometheus
+
+name: vmware
+
+retentionTime: 8h
+
+additionalScrapeConfigs: {}
+
+scrapeInterval: "60s"
+
+configMaps:
+  - atlas-netbox-sd
+
+alertmanagers:
+  hosts:
+    - alertmanager.scaleout.eu-de-1.cloud.sap
```

Comment:
Can you use `alertmanager-internal.scaleout.eu-de-1.cloud.sap` and `alertmanager-internal.scaleout.eu-nl-1.cloud.sap` as we want to enable 2FA for the other endpoints.